src/index.html

```diff
@@ -7,8 +7,63 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="home">
+        <a href="index.html">
+          <img
+            class="logo"
+            src="images/logo.png"
+            alt="MoYo_logotype">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__box">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__box">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__box">
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__box">
+              <a
+                href="#laptops&computers"
+                class="nav__link"
+                data-qa="hover">Laptops & Computers</a>
```

Comment:
Here too. Check the example from the checklist:

![image](https://github.com/mate-academy/layout_moyo-header/assets/23013525/80dc0f83-0edd-46ad-84d0-acd008e2bae8)


```suggestion
              <a
                href="#laptops&computers"
                class="nav__link"
                data-qa="hover"
              >
                Laptops & Computers
              </a>
```